test/rubygems/test_project_sanity.rb

```diff
@@ -12,4 +12,10 @@ def test_manifest_is_up_to_date
     assert status.success?, "Expected Manifest.txt to be up to date, but it's not. Run `rake update_manifest` to sync it."
   end
 
+  def test_require_rubygems_package
+    _, status = Open3.capture2e("ruby -v --disable-gems -I 'lib' -e 'require \"rubygems/package\"'")
```

Comment:
I created this helper `ruby_with_rubygems_in_load_path` to simplify command line building when creating ruby subprocesses. You could use it here.

Also, I would capture the output into a variable and pass it as a second argument to the assert below so that it is displayed in case of error.